### PR TITLE
Better error message for nbfmt

### PR DESCRIPTION
**Pull request by rmlarose**
Follow up from #124. The `nbfmt` error message is the "Cirq error message" which says to run `check/nbformat` which doesn't exist in ReCirq. This PR updates the error message to reference the correct file, namely `dev_tools/nbfmt`. For example:

```bash
The following notebooks require formatting.
- docs/fermi_hubbard/experiment_example.ipynb
Notebooks are not formatted. Please run 'dev_tools/nbfmt --apply'
```

### CODE REVIEW

File: docs/fermi_hubbard/experiment_example.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```
Comment (mpharrigan) [inline docs/fermi_hubbard/experiment_example.ipynb:14]: why is this change included?

**Review by mpharrigan — COMMENTED** [review]
haha awesome

File: docs/fermi_hubbard/experiment_example.ipynb (hunk 1 above)
Comment (rmlarose) [reply docs/fermi_hubbard/experiment_example.ipynb]: Ah, leftover from intentional unformatting. Fixed now.

**Review by mpharrigan — APPROVED** [review]
